docs/Reference/Pantheon-CLI-Syntax.md

```diff
@@ -241,8 +243,28 @@ The default is `false`.
 metrics-host="127.0.0.1"
 ```
 
-Specifies the host on which the [Prometheus](https://prometheus.io/) monitoring service accesses Pantheon
-metrics. The default is `127.0.0.1`. The metrics server respects the [`--host-whitelist` option](#host-whitelist).
+Specifies the host on which [Prometheus](https://prometheus.io/) accesses [Pantheon metrics](../Using-Pantheon/Debugging.md#monitor-node-performance-using-prometheus) or which 
+Pantheon pushes metrics to a [Prometheus Pushgateway](https://github.com/prometheus/pushgateway). 
+The metrics server respects the [`--host-whitelist` option](#host-whitelist).
```

Comment:
This will change with #678  - push hosts and ports will have their own CLI flags.  For this verbiage it is `--metrics-push-host`

For the push mode it may be valuable to note that if using push you almost always will have the push gateway on a different machine so you will almost always want to set --metrics-push-host when --metrics-mode=push is enabled.

Reply:
Done